src/compas_fab/robots/robot.py

```diff
@@ -548,7 +609,27 @@ def get_group_names_from_link_name(self, link_name):
         return group_names
 
     def get_position_by_joint_name(self, configuration, joint_name, group=None):
-        """Returns the value of the joint_name in the passed configuration.
+        """Returns the joint value of the joint with ``joint_name`` in the passed configuration.
+
+        Parameters
+        ----------
+        configuration : :class:`Configuration`
+            The configuration of the configurable joints.
+        joint_name : str
+            Name of joint.
+        group : str, optional
+            The name of the planning group. By default the main planning group.
+
+        Returns
+        -------
+        list of floats
+            Joint values for ``joint_name``.
+
+        Raises
+        ------
+        ValueError
+            If the number of configurable joints in defined group is not equal
+            to the planning group's number of joints.
```

Comment:
Not quite right: If the number of joints in the `configuration` parameter does not match the configurable joints of the given `group` [..].

Reply:
71d5122